src/modm/platform/core/hosted/delay.hpp.in

```diff
@@ -74,19 +77,21 @@ delay_ms(int ms)
 inline void
 delayNanoseconds(int)
 {
+	// impossible to do
 	// TODO
 }
 
 inline void
 delayMicroseconds(int)
 {
+	// impossible to do
```

Comment:
Might be good to implement a stopgap. E.g. if the value is greater than 1ms in microseconds, sleep in milliseconds; maybe also have a busy loop variant for less than 1ms? Or maybe set the threshold lower, so anything above 0.1ms sleeps for 1ms, and everything below is instant?

I'm trying to find a solution which avoids hard-to-find bugs... but it should be something simple, there's no sense in making it more complicated.

Also, this might (?) work as an implementation: `std::this_thread::sleep_for(std::chrono::microseconds(1));`

Reply:
Messed around with this for a bit, but couldn't seem to figure out a good solution. This may be something I ask you to look into later.

Reply:
👌 I figure it's fine to leave it for now and I can come back and take a look; it probably won't be a major issue.